src/script/ceph-release-notes

```diff
@@ -29,6 +29,7 @@ import github
 import os
 import re
 import sys
+import requests
```

Comment:
could you add it to the pip install in the comment at the beginning of the file ?